waspc/src/Parser/Auth.hs

```diff
@@ -14,24 +15,52 @@ auth = do
     L.reserved L.reservedNameAuth
     authProperties <- P.waspClosure (L.commaSep1 authProperty)
 
+    let userEntityPs = [s | AuthPropertyUserEntity s <- authProperties]
```

Comment:
Ps stands for props? Uff it took me some time to figure this out. Why not Props?

Reply:
we are in Haskell come one :D

Reply:
changed to Props